### PR TITLE
Milestone 1 - List structure

### PR DESCRIPTION
**Pull request by Jmagero**

### CODE REVIEW

**Comment by Jmagero** [comment]
- [x] Set up the project webpack.
- [x] Created an index.html file for  HTML markup. Used HTML Webpack Plugin to setup index.html  template
- [x] Added an index.js file and set an array of some simple to do tasks (array of objects)
- [x]  Created a function to iterate over the tasks array and populated an HTML list item element for each task.

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+main {
+  margin: 0 auto;
+  background-color: white;
+  border-top: 20px black solid;
+  border-bottom: 20px black solid;
```
Comment (mricanho) [inline src/style.css]: - [ ] You don't need these borders, check the original design:

image

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @Jmagero, 

Good job so far! 🔥🔥 Your first milestone it's almost done, good commit messages and gitflow work, your  JS code it's awesome.

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [ ] It's super important that you add a README file, this is the way in which anyone could know what is your project and how they can set up the project locally. The instructions are important too.

- [ ] Rework your app's style, you are close but you can do better
Your style:

image

Target style:

image


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Jmagero** [comment]
- [x] Add README file
- [x] Reworked app style

File: src/style.css (hunk 1 above)
Comment (Davidosky007) [reply src/style.css]: * done 👍

**Comment by Davidosky007** [comment]
## Hi @Jmagero 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
nice-job

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**